meetuppizza/tests.py

```diff
@@ -1,6 +1,23 @@
 from django.test import TestCase
+from django.contrib.auth.models import User
+from django.test import Client
+from meetuppizza.forms import RegistrationForm
+from django.contrib import auth
+
+import pdb
+
 
 class Test(TestCase):
+  def setUp(self):
+    self.params =  {
+      'username':'Bjorn', 
+      'email':'bjorn@bjorn.com', 
+      'password1':'bjornbjorn', 
+      'password2':'bjornbjorn'
+    }
+
+    self.client = Client()
```

Comment:
I thought this was already defined on instances of `django.test.TestCase`. Is this new in 1.9, maybe?